### ISSUE TITLE
Support disambiguation of scores from metrics that returns same score names

### ISSUE THREAD

**Issue by yoavkatz** [IBM/unitxt#802]
Today, If metrics, returns the same score (e.g. "f1") then the first metric score is returned, and the second metric is ignored.

    metrics ["metrics.bert_score.distilbert_base_uncased","metrics.bert_score.deberta_base_mnli"],
    
It would be good to be able to add a prefix:

    metrics ["metrics.bert_score.distilbert_base_uncased[score_prefix="distilbert_]","metrics.bert_score.deberta_base_mnli[score_prefix="deberta_]"],
    
  The score prefix will be appended to any score returned by the metrics (except score_name, and score which are reserved names).

**Comment by elronbandel** [comment]
I think @arielge also dealt with it. @arielge if you have more to add to this issue feel free.

**Comment by arielge** [comment]
Indeed, I encountered this issue in the context of overwrite args - https://github.com/IBM/unitxt/issues/737, but nothing to add otherwise
